indico/core/db/sqlalchemy/principals.py

```diff
@@ -523,6 +572,12 @@ def __auto_table_args(cls):
             checks.append(db.CheckConstraint('type != {} OR (NOT full_access AND array_length(permissions, 1) IS NULL)'
                                              .format(PrincipalType.network),
                                              'networks_read_only'))
+        if cls.allow_registration_form:
+            # you can match a network acl entry without being logged in.
+            # we never want that for anything but simple read access
```

Comment:
don't blindly copy&paste comments ;)

Reply:
I forget to change it, this is draft only ;)